modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -36,35 +36,151 @@ Showcase apps demonstrate many features of {product-name}. To see these features
 +
 For more information about features, see xref:identity-management.adoc[{keycloak-service} Service]
 
-
 . To demonstrate {unifiedpush-service}, xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
 
-. To demonstrate {sync-service}:
-+
-NOTE: You can only demonstrate {sync-service} on Cordova.
```

Comment:
There's still an upstream restriction here,
NOTE: You can only demonstrate {sync-service} if you are building a JavaScript app

@wtrocki am i correct in this?

Reply:
https://github.com/aerogear/mobile-docs/pull/682/commits/15461598bc02a084758e5362f9a72b12863a913e